cncserver.js

```diff
@@ -1125,6 +1126,18 @@ function serialPortReadyCallback() {
       return;
     }
 
+    // Setting the value of the power to the pen
+    if (typeof inPen.power !== "undefined") {
+      powers = botConf.get('penpower');
+      if(typeof powers === "undefined") { // We have no super powers
+        powers = {min: 0, max: 0};  //Set the powers to zero
+      }
+
+      run('custom', cmdstr('penpower', {p: Math.round(inPen.power * powers.max) + Number(powers.min)}));
+      pen.power = inPen.power;
+      if (callback) callback(true);

```

Comment:
Something we missed on the floor of LAX: We should very likely leave the function at this point with a `return;`, right after the callback is called of course.